src/game/console_cmd.c

```diff
@@ -182,7 +182,8 @@ static COMMAND_RESULT Console_Cmd_Teleport(const char *const args)
              item_num++) {
             const ITEM_INFO *const item = &g_Items[item_num];
             const bool is_pickup =
-                Object_IsObjectType(item->object_id, g_PickupObjects);
+                Object_IsObjectType(item->object_id, g_PickupObjects)
+                || item->object_id == O_SAVEGAME_ITEM;
```

Comment:
This should be tracked in a separate issue + changelog entry. I suggest amending the condition below to spell `(is_pickup || item->object_id == O_SAVEGAME_ITEM)`, or rename this variable to something that reflects its purpose better (`is_consumable` for example).